Updated download link to Docker Toolbox

codingwithcookie opened the pull request:
The old get docker link no longer has the toolbox listed on the page, it has been moved to a github repo.

azurecla commented:
Hi __@antonfrMSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

alancameronwills commented:
Editors, please accept.

ggailey777 commented:
@alancameronwills can you work with @antonfrMSFT to fix the merge conflicts in the PR, or if you don't need it anymore please close it.

ggailey777 commented:
Closing this due to unresolved merge conflict. If you still need this fix in the public repo, please reopen the PR and fix the conflict.

codingwithcookie committed:
Updated download link to Docker Toolbox

The old get docker link no longer has the toolbox listed on the page, it has been moved to a github repo.
